Functional.py

```diff
@@ -0,0 +1,89 @@
+
+# # 1
+
+def sequential_map(*args):
+    l = len(args) - 1
+    c = args[-1]
+    for i in args[0:l]:
+        c = i(c)
+    return list(c)
```

Comment:
Переданные функции должны применяться ко всем значениям в контейнере, а не к самому контейнеру
```suggestion
    *functions, container = args
    for func in functions:
        container = map(func, container)
    return list(container)
```